parlai/tasks/friends/agents.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Optional
+from parlai.core.opt import Opt
+from parlai.core.teachers import DialogTeacher
+from parlai.core.params import ParlaiParser
+from .build import build
+
+import copy
+import os
+import json
+
+
+def _path(opt, *additions):
+    return os.path.join(
+        opt['datapath'], 'Friends', 'friends-corpus/utterances.jsonl', *additions
+    )
+
+
+class DefaultTeacher(DialogTeacher):
+    START_TOKEN = '<START>'
```

Comment:
For these constants, maybe it makes sense to set them via flags, that way they can be consistent when we are mixing them with other datasets that have similar tokens, See [this](https://github.com/facebookresearch/ParlAI/blob/7660197631c06ee41c0ce25c5de37a74b11ecb77/projects/blenderbot2/agents/blenderbot2.py#L279-L284).

Also, in most of our recent projects we prefer a different style for the special tokens. See the silence token [here](https://github.com/fairinternal/ParlAI-Internal/blob/1ed6496e549944cf02017930a63baa7555f9055a/projects/blenderbot3/tasks.py#L121).